A32NX/html_ui/Pages/A32NX_Utils/NXFMGCFlightPhases.js

```diff
@@ -0,0 +1,10 @@
+const FMGC_FLIGHT_PHASES = {
+    PREFLIGHT: 0,
+    TAKEOFF: 1,
+    CLIMB: 2,
+    CRUISE: 3,
+    DESCENT: 4,
+    APPROACH: 5,
+    GOAROUND : 6,
+    DONE: 7
+};
```

Comment:
While we do not have real (Typescript) enums here, something irks me about the double uppercase, it looks unusual in the code everywhere.

I think I'd expect either UPPER_CASE.CamelCase, or CamelCase.UPPERCASE. Do we have prior art for other constants like this?

Reply:
`CamelCase.UPPERCASE` looks right to me.

Reply:
I'll change that